### PR TITLE
litespi/modules: add MX25L6406E and W25Q512JV to non-generated modules

### LINKED ISSUE #17: Add non-generated SPI NOR chip example
Currently have https://github.com/litex-hub/litespi/blob/8075781f3be4f783ce5c0af4a2a296e4ed3e224b/litespi/modules/modules.py#L5

We should have **at least one** non-generated SPI NOR chip example in the code to make sure that it actually works.

### PR DESCRIPTION
**Pull request by kowalewskijan**
Fixes #17 
Docs: [MX25L6406E](https://www.macronix.com/Lists/Datasheet/Attachments/7370/MX25L6406E,%203V,%2064Mb,%20v1.9.pdf), [W25Q512JV](https://www.winbond.com/resource-files/W25Q512JV%20SPI%20RevB%2006252019%20KMS.pdf)

### CODE REVIEW

File: litespi/modules/modules.py [hunk 1]
```diff
@@ -3,3 +3,52 @@
 from litespi.ids import SpiNorFlashManufacturerIDs
 
 # Define non-generated SPI NOR chips here
+
+class MX25L6406E(SpiNorFlashModule):
+
```
Comment (mithro) [inline litespi/modules/modules.py:8]: Can you add a link to the datasheets in this module?

**Comment by mithro** [comment]
Please link to the datasheets in the docs.

File: litespi/modules/modules.py (hunk 1 above)
Comment (kowalewskijan) [reply litespi/modules/modules.py:8]: This should be rather a triple quoted docstring or a comment above each class?

Comment (kowalewskijan) [reply litespi/modules/modules.py:10]: I added them as comments above each class (direct links to pdfs).

File: litespi/modules/modules.py [hunk 2]
```diff
@@ -3,3 +3,56 @@
...
+    supported_opcodes = [
+        SpiNorFlashOpCodes.READ_1_1_1,
+        SpiNorFlashOpCodes.READ_1_1_1_FAST,
+        SpiNorFlashOpCodes.READ_1_1_2,
+        SpiNorFlashOpCodes.PP_1_1_1,
+    ]
+    dummy_bits = 8
+
+
+# W25Q512JV Datasheet:
```
Comment (mithro) [inline litespi/modules/modules.py:28]: These should be in the docstrings so they appear in the generated documentation output.

**Comment by kowalewskijan** [comment]
I added missing docstrings to each class. Can we merge it? @mithro

**Pull request merged**